docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -1118,10 +1120,62 @@ will be created together with the autogenerated ones.
 
 * Access rights: XROAD_SYSTEM_ADMINISTRATOR and XROAD_SERVICE_ADMINISTRATOR
 
-Access rights for a single endpoint can be add with:
+List service endpoints can be done with:
 ```
-xrdsst endpoint add-access
+xrdsst endpoint list --ss <SECURITY_SERVER_NAME>
```

Comment:
Something has gone wrong here, you have placed "endpoint list" documentation over "endpoint add-access" and "cert list" under "endpoint list".